Tools/CMake/AMReXParallelBackends.cmake

```
@@ -210,6 +210,10 @@ if (AMReX_HIP)
    find_package(rocrand REQUIRED CONFIG)
    find_package(rocprim REQUIRED CONFIG)
    find_package(hiprand REQUIRED CONFIG)
+   if (AMReX_ROCTX)
+       target_include_directories(amrex PUBLIC ${HIP_PATH}/../roctracer/include ${HIP_PATH}/../rocprofiler/include)
```

Just to keep taps:
```suggestion
       # To be modernized in the future, please see:
       # https://github.com/ROCm-Developer-Tools/roctracer/issues/56
       target_include_directories(amrex PUBLIC ${HIP_PATH}/../roctracer/include ${HIP_PATH}/../rocprofiler/include)
```

We could use [find_library](https://cmake.org/cmake/help/latest/command/find_library.html) and [find_file](https://cmake.org/cmake/help/latest/command/find_file.html), but I think there is atm no reason to over-engineer this.